### ISSUE TITLE
Unable to delete locations

### ISSUE THREAD

**Issue by goetzk** [snipe/snipe-it#1017]
When trying to delete a location I'm getting 'Server Error: 500 (Internal Server Error)'.

apache2handler log records
```
[2015-08-05 00:22:01] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function count() on a non-object' in /srv/snipe-it/app/controllers/admin/LocationsController.php:192
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
```

I tested the following situations:
- location with asset
- location without assets
- location as a top level
- location as a sub location

**Comment by snipe** [comment]
What version/branch?

**Comment by goetzk** [comment]
sorry, we're on 2.0-pre-beta-5-g1c2a409

**Comment by snipe** [comment]
The 2.0 series isn't stable, that's why it's a pre-release. You can try download the latest from the 2.0 branch, but no guarantees it will be better. We're still working through issues. That said, I cannot reproduce this locally at all.

**Comment by madd15** [comment]
I do believe this is fixed on the current 2.0 branch

**Comment by madd15** [comment]
#970

**Comment by snipe** [comment]
I thought so too. Thanks

**Comment by goetzk** [comment]
On 5/08/2015 12:00, madd15 wrote:
> I do believe this is fixed on the current 2.0 branch
thanks!
I will do an upgrade soon to check.
